pkg/controllers/nodeclaim/lifecycle/registration.go

```diff
@@ -80,6 +80,10 @@ func (r *Registration) Reconcile(ctx context.Context, nodeClaim *v1beta1.NodeCla
 
 func (r *Registration) syncNode(ctx context.Context, nodeClaim *v1beta1.NodeClaim, node *v1.Node) error {
```

Comment:
Are there any updates to the registration_test.go file that need to happen with this change?

Reply:
Added.